src/HanziWriter.js

```diff
@@ -12,6 +12,7 @@ const { assign, isMSBrowser, timeout } = require('./utils');
 
 const defaultOptions = {
   charDataLoader: defaultCharDataLoader,
+  onCharDataLoaderError: null,
```

Comment:
nit: could this be called `onLoadCharDataError` instead? That way it looks more consistent with `isLoadingCharData`.

Reply:
Of course.